sbi/mcmc/init_strategy.py

```diff
@@ -57,6 +61,7 @@ def sir(
         init_param_candidates = []
         for i in range(sir_num_batches):
             batch_draws = prior.sample((sir_batch_size,)).detach()
+            batch_draws, log_abs_det = transform.inverse(batch_draws)
             init_param_candidates.append(batch_draws)
             log_weights.append(potential_fn(batch_draws.numpy()).detach())
```

Comment:
Must also take `log_abs_det` into account? Or is this inherent in the `potential_fn`?